cloudquery/sdk/scheduler/scheduler.py

```diff
@@ -103,11 +104,13 @@ def resolve_table(
                 )
             total_resources = 0
             for item in resolver.resolve(client, parent_item):
+                print("item", item)
```

Comment:
You have some print()s dotted around here

Reply:
fixed 👍